tests/test_build.py

```diff
@@ -604,6 +595,27 @@ def test_analytics(sphinx_build_factory, provider, tags):
     assert tags_found is True
 
 
+def test_plausible(sphinx_build_factory):
+    provider = {
+        "html_theme_options.analytics": {
+            "plausible_analytics_domain": "toto",
+            "plausible_analytics_url": "http://.../script.js",
+        }
+    }
+    attr = ["data-domain", "toto"]
+    confoverrides = provider
+    sphinx_build = sphinx_build_factory("base", confoverrides=confoverrides)
+    sphinx_build.build()
+    index_html = sphinx_build.html_tree("index.html")
+
+    # Search all the scripts and make sure one of them has the Google tag in there
+    attr_found = False
+    for script in index_html.select("script"):
+        if script.attrs.get(attr[0]) == attr[1]:
+            attr_found = True
+    assert attr_found is True
```

Comment:
```suggestion
    confoverrides = provider
    sphinx_build = sphinx_build_factory("base", confoverrides=confoverrides)
    sphinx_build.build()
    index_html = sphinx_build.html_tree("index.html")

    # Search all the scripts and make sure one of them has the Google tag in there
    attr_found = False
    for script in index_html.select("script"):
        if script.attrs.get("data-domain") == "toto":
            attr_found = True
    assert attr_found is True
```

Reply:
I feel like this is a bit clearer so you don't have to look up what `attrs` means. That work?